Assets/Mirror/Runtime/Transport/Websocket/ClientJs.cs

```diff
@@ -40,6 +40,8 @@ public bool IsConnected
         int m_NativeRef = 0;
         readonly int id;
 
+        static byte[] buffer = new byte[1500];
```

Comment:
does the websocket transport have a MaxPacketSize similar to LLAPI/Telepathy?
for telepathy I will do buffer = new buffer[MaxPacketSize] etc. which would avoid the code below

Reply:
in fact, there should be some kind of max size to avoid allocation attacks where an attacker might try to send a header that says '10gb packet incoming', causing the server to run out of memory

Reply:
Nope. The max size of a WebSockets "frame" is petabytes.

Reply:
perhaps limit it yourself then?
Telepathy's theoretical max size is int.max too, but I added a limit that can be configured in the component:
![image](https://user-images.githubusercontent.com/16416509/56883769-f4a2da80-6a67-11e9-89b2-ecf08fb2017b.png)

this is very important to have. all games using our websockets implementation are vulnerable to allocation attacks otherwise.